installer/charts/rhtap-openshift/templates/projects.yaml

```diff
@@ -15,6 +15,13 @@ apiVersion: project.openshift.io/v1
 kind: ProjectRequest
 description: {{ $p }}
 displayName: {{ $p }}
+metadata:
```

Comment:
I'm fine with merging that if it works, but I thought creating `openshift-*` namespaces/projects was sometimes messing up with some operators deployment.

Reply:
It does not work with Project Request cause we have some policies in OpenShift to block, but should be fine when we create as a namespace. This is required cause openshift-storage Project is create by ODF if done from UI but not when using subscription CR. This is working fine at the moment.